pipeline_steps/checkmarx.groovy

```diff
@@ -80,6 +80,10 @@ def scan(String scan_type, String repo_name, String exclude_folders){
                 // exception must propagate back to the retry call
                 throw e
             } //try
+
```

Comment:
This following block is within the retry block so it's going to cause unintentional retries when there is a genuine failure.

Reply:
Yes, will move below the retry logic ✅